build-system/npm-publish/write-package-files.js

```diff
@@ -23,6 +23,17 @@ async function shouldSkip() {
   }
 }
 
+/**
+ * Determines whether a given directory path contains CSS files
+ *
+ * @param {string} path Path to a directory.
+ * @return {Promise<boolean>}
+ */
+async function hasStylesheets(path) {
+  const files = await readdir(path);
+  return files.some((file) => file.endsWith('.css'));
```

Comment:
We could also use `fast-glob` which we use elsewhere for similar things:
```js
const fastGlob = require('fast-glob');

....

// Returns true if an extension exports CSS files.
function hasStylesheets(extension, version) {
  return fastGlob.sync(`extensions/${extension}/${version}/dist/**/*.css`).length > 0
}


```

Reply:
Cool, I didn't know that package was being used already. Makes sense to me 👍 

Is the preference to use the sync or the Promise-based API though?

Reply:
I'd likely use the `async` version here since it would be a smaller change from where you currently are.

Reply:
Ah, I should have used `async` in my example...I got lazy